backend/pkg/rest/applications_resources_controller.go

```diff
@@ -71,3 +73,162 @@ func (s *Server) GetApplicationResources(params operations.GetApplicationResourc
 			Total: &total,
 		})
 }
+
+func (s *Server) DeleteApplicationResource(id string) middleware.Responder {
+	// Get application resource info.
+	resource, err := s.dbHandler.ResourceTable().GetDBResource(id, false)
+	if err != nil {
+		log.Errorf("application resource not found by id %q: %v", id, err)
+		return operations.NewDeleteApplicationResourcesIDNotFound()
+	}
+
+	// Get package IDs that has relations with the resource.
+	relatedPackageIDs, err := s.getRelatedPackageIDs(resource.ID)
+	if err != nil {
+		log.Errorf("Failed to get packages for resource ID=%s: %v", resource.ID, err)
+		operations.NewDeleteApplicationResourcesIDNotFound()
```

Comment:
`operations.NewDeleteApplicationResourcesIDNotFound()`?
Not sure that is the return we need, also missing a `return` call, no?